manim/scene/scene.py

```diff
@@ -876,8 +876,26 @@ def add_sound(self, sound_file, time_offset=0, gain=None, **kwargs):
             The offset in the sound file after which
             the sound can be played.
 
-        gain :
+        gain : Amplification of the sound.
 
+        Examples
+        --------
+        .. manim:: SoundExample
+            :save_last_frame:
+
+            class SoundExample(Scene):
+                # Source of sound under Creative Commons 0 License. https://freesound.org/people/Druminfected/sounds/250551/
+                def construct(self):
+                    dot = Dot().set_color(GREEN)
+                    self.add_sound("/docs/source/_static/click.wav")
```

Comment:
I dont know if this is the exact path syntax for the docs, @behackl could you have a look on this?